tests/add_user.spec.ts

```diff
@@ -45,8 +52,10 @@ test.describe("Add User functional", () => {
     const addUserPage = new AddUserPage(page);
     const generator = new Generator();
 
-    await addUserPage.setGender(generator.getGender());
-    await addUserPage.setUserName(generator.getName(false), true);
+    const user = new UserDTO(generator.getGender(), generator.getName(false));
+
+    await addUserPage.setGender(user.getGender());
+    await addUserPage.setUserName(user.getUsername(), true);
```

Comment:
Would there even be a situation when you don't want to blur the filed after filling it? If not, remove the blur argument, and make the filed blur always